README.md

```diff
@@ -91,6 +91,12 @@ Then, run the following commands to copy the library over:
 
 `sudo cp ../src/SimpleAmqpClient/*.h /usr/local/include/SimpleAmqpClient`
 
+5) Configure linker
+
+After adding the library files to /usr.local/lib. The linker needs to be configure with: 
```

Comment:
Typo in the filepath & on configured. 
It might be better to say `After adding the blah blah. Configure the dynamic linker with SimpleAmqpClient with:`

Reply:
Path should also be in backticks. ` so it becomes formatted